gobblin-restli/server.gradle

```diff
@@ -78,5 +78,4 @@ artifacts {
 }
 
 war {
-  classifier "war"
```

Comment:
I think this is needed in general (you just needed to remove it for running from IDE).